service/src/main/java/uk/gov/hmcts/reform/fpl/model/PolicyDatas.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import uk.gov.hmcts.reform.ccd.model.OrganisationPolicy;
+import uk.gov.hmcts.reform.fpl.enums.CaseRole;
+import uk.gov.hmcts.reform.fpl.utils.PolicyHelper;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+public interface PolicyDatas {
```

Comment:
Think PolicyData is a plural form already. Should it be renamed to PolicyData instead?

Reply:
haha, ok renamed